cypress/e2e/publish.cy.ts

```diff
@@ -1,25 +1,93 @@
 describe("Publish", () => {
   it("creates an episode and publishes it to a feed", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+
+    // Login
     cy.visit(`https://${Cypress.env("ID_HOST")}/`);
     cy.contains("Sign in");
     cy.get("#login").type(Cypress.env("PUBLISH_USER"));
     cy.get("#password").type(Cypress.env("PUBLISH_PASS"));
     cy.get(".btn.submit").click();
 
+    // Confirm login
     cy.contains("Welcome, test test");
 
-    const now = new Date().toISOString();
-
+    // Create a new series
     cy.visit(`https://${Cypress.env("PUBLISH_HOST")}/`);
     cy.get('a[href="/series/new"]').click();
-    cy.get("#title").type(`Acceptance Test: ${now}`);
-    cy.get("#shortDescription").type(`Acceptance Test: ${now}`);
-    // cy.get('prx-button[green=1] button.green').click();
-
-    // cy.wait(4000)
-    // cy.url().then((url) => {
-    //   const seriesId = url.split('/').at(-1);
-    //   cy.log(seriesId);
-    // });
+    cy.get("#title").type(canary);
+    cy.get("#shortDescription").type(canary);
+    cy.get("prx-button[green=1] button.green").click();
+
+    // Confirm series creation
+    // This button shows up once we've moved from /new to the series show page
+    cy.get("button.delete").should("exist");
+
+    cy.url().then((seriesUrl) => {
```

Comment:
No async/await?

Reply:
Cypress, as far as I can tell, can't use await in that scenario. But I didn't dig into it too much, I could have been looking at old docs/examples. But I saw zero examples of async tests.